utils/changelog.js

```diff
@@ -9,11 +9,26 @@ const groupLog = require('./changelog/group');
 process.chdir(join(__dirname, '..'));
 
 async function getLatestStableTag() {
+  const headers = {};
+
+  const token = process.env.GITHUB_PAT;
```

Comment:
My instinctual naming would be `GITHUB_TOKEN`, but not a huge deal.

Reply:
That is better. You can use different types of tokens and this will still work. I updated it.